src/library/scala/collection/immutable/Vector.scala

```diff
@@ -466,24 +466,24 @@ override def companion: GenericCompanion[Vector] = Vector
       display5 = copyRange(display5, oldLeft, newLeft)
   }
 
-  private def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
+  private[this] def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
```

Comment:
@lrytz - Yeah, I see that.  It does (or has--not sure about now) occasionally make a difference with fields and it was faster for me to check the benchmark than the bytecode.  But it seems there was just a benchmarking fluctuation that made everything I did after the one bad benchmark seem like it was helping (a tiny bit).